src/Exception/ErrorException.php

```diff
@@ -149,6 +149,6 @@ private function isXdebugStackAvailable(): bool
         }
 
         // Xdebug 3 and later, proper mode is required
-        return str_contains(ini_get('xdebug.mode'), 'develop');
+        return str_contains(ini_get('xdebug.mode'), 'debug');
```

Comment:
`mode` [docs](https://xdebug.org/docs/all_settings#mode):

```
develop Enables Development Helpers...
```

`xdebug_get_function_stack()` is included in [development helpers](https://xdebug.org/docs/develop).

-----

What warning are you showing?

Reply:
![image](https://github.com/user-attachments/assets/3937f736-0f98-44c7-adbe-fcc602d8aac2)